sdk/bootstrap/main.go

```diff
@@ -130,13 +135,28 @@ func splitAndClean(s string, sep string) []string {
 }
 
 // env print
-
 func printEnv() {
-	env := os.Environ()
+	env := logSensitiveEnviron()
 	sort.Strings(env)
 	log.Printf("Bootstrapping with environment:\n%s", strings.Join(env, "\n"))
 }
 
+func logSensitiveEnviron() (ret []string) {
```

Comment:
The name of this function is misleading. It is not logging anything, but is filtering the environment.